hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -937,13 +937,6 @@ private int getWhenNoSSLDetected() {
     public static final HazelcastProperty INIT_CLUSTER_VERSION
             = new HazelcastProperty("hazelcast.init.cluster.version");
 
-    /**
```

Comment:
@mdogan you don't need to review the whole PR but just to let you know I decided to remove the legacy member list format, hope that's ok.

Reply:
sure 👍